.gitignore

```diff
@@ -1 +1,2 @@
 /build
+.vscode
```

Comment:
This change should be in different pull request or at least in a separate commit.